data/data.json

```diff
@@ -95,5 +95,11 @@
     "link": "https://openbazaar.org",
     "categories": ["E-commerce","Shopping"],
     "crypto": ["BTC"]
+  },
+  {
+    "title": "Namecheap",
+    "link": "https://www.namecheap.com",
+    "categories": ["Domain name registration","Web Hosting"],
```

Comment:
Could we change to `Domain Registration`?

Reply:
Sure thing!